.github/workflows/docker_build_images.yml

```diff
@@ -26,13 +94,13 @@ jobs:
 
     steps:
       - name: Check out code
-        uses: actions/checkout@v4
+        uses: actions/checkout@692973e3d937129bcbf40652eb9f2f61becf3332 # v4.1.7
```

Comment:
What is the reason to use the SHA here instead of a version?

Reply:
Using the SHA instead of the version is part of the OpenSSF best practices. We started doing this some time ago through https://github.com/vitessio/vitess/pull/16612. This is to comply with https://github.com/vitessio/vitess/issues/16607.